vsp/vsp.go

```diff
@@ -126,7 +126,7 @@ func New(hostname, pubKeyStr string, purchaseAccount, changeAccount uint32, dial
 						log.Infof("VSP has successfully confirmed the fee tx for %v", txHash)
 					case "error":
 						log.Warnf("VSP failed to broadcast feetx for %v -- restarting process", txHash)
-						v.Queue(ctx, *txHash, nil)
+						v.Queue(ctx, *txHash, feeTx)
```

Comment:
I think this is wrong; this will retry with the same `feeTx`.  The nil retries by generating a new feetx.